commands.py

```diff
@@ -96,32 +96,58 @@ def run(self, _: sublime.Edit) -> None:
 class CopilotAcceptSuggestionCommand(CopilotTextCommand):
     @_provide_session()
     def run(self, session: Session, edit: sublime.Edit) -> None:
-        completion = Completion(self.view)
-        if not completion.is_visible:
+        completion_manager = ViewCompletionManager(self.view)
+        if not completion_manager.is_visible:
             return
 
-        completion.hide()
+        completion_manager.hide()
+
+        completion = completion_manager.current_completion
+        if not completion:
+            return
 
         # Remove the current line and then insert full text.
         # We don't have to care whether it's an inline completion or not.
-        source_line_region = self.view.line(completion.region[1])
+        source_line_region = self.view.line(completion["positionSt"])
         self.view.erase(edit, source_line_region)
-        self.view.insert(edit, source_line_region.begin(), completion.text)
+        self.view.insert(edit, source_line_region.begin(), completion["text"])
+
+        # notify the current completion as accepted
+        self._record_telemetry(session, REQ_NOTIFY_ACCEPTED, {"uuid": completion["uuid"]})
 
-        # TODO: When a suggestion is accept, we need to send a REQ_NOTIFY_REJECTED
-        # request with all other completions which weren't accepted
-        self._record_telemetry(session, REQ_NOTIFY_ACCEPTED, {"uuid": completion.uuid})
+        # notify all other completions as rejected
+        other_uuids = [completion["uuid"] for completion in completion_manager.completions]
+        other_uuids.remove(completion["uuid"])
+        if other_uuids:
```

Comment:
Do we really need to mark other `uuid`s as rejected?

Reply:
@TheSecEng since this is his comment TODO.

Reply:
See above comment

Reply:
Move discussion to https://github.com/TheSecEng/LSP-copilot/pull/13#discussion_r922831886